src/store/index.ts

```diff
@@ -236,9 +241,22 @@ const resolvers: Resolvers = {
             })
         }
     },
-
+    Recipient: {
+        name: async (recipient: Recipient, _context): Promise<String> => {
```

Comment:
so... you don't actually need resolvers if the recipient object has these fields... totally fine to leave.